tests/app/views/test_buyers.py

```diff
@@ -1874,6 +1880,394 @@ def test_404_if_lot_does_not_allow_brief(self, data_api_client):
         assert res.status_code == 404
 
 
+class TestDownloadBriefResponsesView(BaseApplicationTest):
```

Comment:
While testing the callable directly seems like a useful approach, I think we're missing some things that we usually test for the view.

We no longer have a test checking that URL maps correctly to the view arguments. And we no longer have a test to check that user authentication is required. I think both are useful for the view, so I think we should add them.